<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Road2TraffLWizardPage</class>
 <widget class="QWizardPage" name="Road2TraffLWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,4">
       <item>
        <widget class="QLabel" name="label_head_road_count">
         <property name="text">
          <string>No. of Roads: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_val_road_count"/>
       </item>
       <item>
        <widget class="QLabel" name="label_head_tl_count">
         <property name="text">
          <string>No. of Traffic Lights: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_val_tl_count"/>
       </item>
       <item>
        <widget class="QLabel" name="label_tl_update">
         <property name="text">
          <string>No. of added/removed Traffic Lights: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_tl_update"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
